src/script/components/InputBar/InputBar.tsx

```diff
@@ -395,13 +394,16 @@ export const InputBar = ({
     if (isEditing) {
       void sendMessageEdit(text, mentions);
     } else {
-      sendMessage(text, mentions);
+      sendTextMessage(text, mentions);
     }
 
-    lexicalRef.current?.focus();
+    editorRef.current?.focus();
+    editorRef.current?.update(() => {
+      $getRoot().clear();
+    });
```

Comment:
This one doesn't work ? 🤔 

https://lexical.dev/docs/api/classes/lexical.LexicalEditor#focus

```suggestion
    editorRef.current?.focus(() => {
      $getRoot().clear();
    });
```

Reply:
oh, good call, let me try 👍

Reply:
Nope, this doesn't work, you need to call `update` in order to be able to update the editor

```
Lexical.dev.js:8973 Uncaught Error: Unable to find an active editor state. State helpers or node methods can only be used synchronously during the callback of editor.update() or editorState.read().
```